opencog/reasoning/RuleEngine/rules/pln/inheritance-to-member.scm

```diff
@@ -0,0 +1,75 @@
+; =============================================================================
+; InheritanceToMemberRule
+;	InheritanceLink( B (SatisfyingSetLink (Variable $X
+; 		(EvaluationLink (pred D (ListLink X C))))))
+;			becomes
+;	MemberLink( B (SatisfyingSetLink ( Variable $X 
+;		(EvaluationLink (pred D (ListLink X C))))))
+; -----------------------------------------------------------------------------
```

Comment:
Is this really what the Inheritance-to-Member rule does?  The python version seems to be more basic:

```
InheritanceLink(Jade robot) => MemberLink(Jade robot).
```